test/mero_SUITE.erl

```diff
@@ -449,20 +451,24 @@ mcas(_) ->
 
 state_ok(_) ->
     State = mero:state(),
+    %% See comments on the .hrl
+    LinksCluster2 = ?LINKED_PROCESSES(3),
     ?assertEqual([{connected, 1},
```

Comment:
There is an alternative, since we don't really care about the actual value of `links` there.
Instead of adding an hrl file and a macro, why don't we just do the following?

```erlang
    ?assertMatch([{connected, 1},
                  {connecting, 0},
                  {failed, 0},
                  {free, 1},
                  {links, 3},
                  {links, _},
                  {message_queue_len, 0},
                  {monitors, 0}],
                 lists:sort(
                     proplists:get_value(cluster2, State))),
```

You can still put a `@todo` to restore the appropriate value once we move past OTP25.
You can even put a note with a link to the interval timers thing that changed and/or open an issue to request the removal of `links`, `message_queue_len`, and `monitors` from `mero_pool:state/1`, in favor of a single stat value: `pid`, so people can just use `process_info/2` on them if they want to get that data. It seems more reasonable to me, since those are technically not _mero_ stats, anyway.

Reply:
I very mucho like that! :)